prov/cxi/include/cxip_faults.h

```diff
@@ -0,0 +1,148 @@
+/*
+ * SPDX-License-Identifier: GPL-2.0
+ *
+ * Copyright (c) 2019 Hewlett Packard Enterprise Development LP
+ */
+
```

Comment:
Missing  `#ifndef ... #define ...  #endif` protection?

Reply:
There is an internal PR that I'll pull out which address that. It was discovered with RHEL 9.3 testing. 

A note for us though -- RHEL 9.3 default LDFLAGS includes -flto=auto, which breaks/strips symbols from the static library generated during RPM builds. This doesn't affect local builds (configure -> make -> make install) because the RPM macros from RHEL aren't picked up in that workflow.

Reply:
Are you going to add the `#ifdef` here? It's a simple change.